### PR TITLE
FileHash TI IOC matching to Security Event as well as Common Security…

### PR DESCRIPTION
**Pull request by aprakash13**
… Log

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/ThreatIntelligenceIndicator/FileHashEntity_CommonSecurityLog.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  | where Active == true
+  | where isnotempty(FileHashValue)
+  |  join (
+     CommonSecurityLog | where TimeGenerated >= ago(dt_lookBack)
+     | where isnotempty(FileHash)
+     | extend CommonSecurityLog_TimeGenerated = TimeGenerated
+  )
+  on $left.FileHashValue == $right.FileHash
+  | summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId
+  | project LatestIndicatorTime, Description, ActivityGroupNames, IndicatorId, ThreatType, ExpirationDateTime, ConfidenceScore, CommonSecurityLog_TimeGenerated, SourceIP, SourcePort, DestinationIP, DestinationPort
```
Comment (shainw) [inline Detections/ThreatIntelligenceIndicator/FileHashEntity_CommonSecurityLog.yaml]: We should bring through DeviceName, DeviceAction, ApplicationProtocol, SourceUserID, DestinationUserID, RequestURL.   These show up in PAN-OS.

Comment (shainw) [reply Detections/ThreatIntelligenceIndicator/FileHashEntity_CommonSecurityLog.yaml]: Also bring through Activity

File: Detections/ThreatIntelligenceIndicator/FileHashEntity_CommonSecurityLog.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  | where isnotempty(FileHashValue)
+  |  join (
+     CommonSecurityLog | where TimeGenerated >= ago(dt_lookBack)
+     | where isnotempty(FileHash)
+     | extend CommonSecurityLog_TimeGenerated = TimeGenerated
+  )
+  on $left.FileHashValue == $right.FileHash
+  | summarize LatestIndicatorTime = arg_max(TimeGenerated, *) by IndicatorId
+  | project LatestIndicatorTime, Description, ActivityGroupNames, IndicatorId, ThreatType, ExpirationDateTime, ConfidenceScore, CommonSecurityLog_TimeGenerated, SourceIP, SourcePort, DestinationIP, DestinationPort 
+  | extend timestamp = CommonSecurityLog_TimeGenerated, IPCustomEntity = SourceIP
```
Comment (shainw) [inline Detections/ThreatIntelligenceIndicator/FileHashEntity_CommonSecurityLog.yaml]: If we bring through the items above, then we can extend HostCustomEntity = DeviceName, AccountCustomEntity = SourceUserID

File: Detections/ThreatIntelligenceIndicator/FileHashEntity_CommonSecurityLog.yaml (hunk 1 above)
Comment (aprakash13) [reply Detections/ThreatIntelligenceIndicator/FileHashEntity_CommonSecurityLog.yaml]: Done. Thanks Shain.

**Pull request merged**